data/random-teams.ts

```diff
@@ -1237,20 +1243,33 @@ export class RandomTeams {
 			return hasMove['dragondance'];
 		case 'Sheer Force':
 			return (!counter.sheerforce || hasAbility['Guts']);
+		case 'Shell Armor':
+			return (species.id === 'omastar' && (hasMove['spikes'] || hasMove['stealthrock']));
```

Comment:
Can we split the changes that affect the main format into their own PR/commit?

Reply:
oh, I'm told that Omastar never gets shell armor in other formats

Reply:
ya